microovn/cmd/microovn/cluster_remove.go

```diff
@@ -2,7 +2,6 @@ package main
 
 import (
 	"context"
-
```

Comment:
Looking at other files, we seem to use a single blank line between standard go libraries and 3rd parties. This line shouldn't be removed